R/article.R

```diff
@@ -256,6 +256,16 @@ informs_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
   )
 }
 
+#' @section `iop_article`: Format for creating submissions to
+#'   IOP journals. Adapted from \samp{https://publishingsupport.iopscience.iop.org/questions/latex-template/}.
```

Comment:
The guidelines provided in the link have lots of information in  `IOPLaTeXGuidelines.tex` example (and its pdf version) following your link. Should we precise in the help page here to look into the Guidelines there before submission ? 
Or the content could be included into the skeleton but this would duplicated. 

Just trying to simplify the step for users looking to use **rticles** for submissions and that won't look at the website expecting our template to be the only thing they need.

Reply:
Added request to read guidelines carefully in the help file and in the template.

Reply:
Thanks